roles/edpm_neutron_dhcp/molecule/default/molecule.yml

```diff
@@ -4,7 +4,7 @@ dependency:
   options:
     role-file: collections.yml
 driver:
-  name: podman
+  name: delegated
```

Comment:
(please bear with me since I am not knowledgeable about molecule) does it switch to a driver that won't allow to run the role locally in podman? rationale? can't we run the agent inside the podman container?

Reply:
Exactly. There may be a way to run this test in a containerized environment. But from experience I can say that tests that require installation of packages and kernel modules are rarely worth the effort, if it's even possible in the first place.

Reply:
I believe @slawqo is able to run these in podman, incl. creating network namespaces etc. I'd prefer this stays this way for easier development.

Reply:
yes, I was able to even run sidecar container tests in https://github.com/openstack-k8s-operators/edpm-ansible/pull/394 using podman driver. And with podman driver is much easier to run molecule tests locally also which helps a lot with development. So unless there is any really important reason to switch to the delegated driver, I would prefer to stay with podman driver here for as long as it's possible.

Reply:
Yes, but running it locally with podman means you have to give it root permissions. Which in my opinion is a no go, as it can cause changes to a dev machine and potentially contaminate environment for further testing.

Not to mention the damage it can potentially cause.

Reply:
yes, we discussed that already on Slack. I opened Jira ticket for that and will try to fix it in next weeks.